app/src/main/java/com/tiptopgoodstudio/androidresources/ui/ResourceListActivity.java

```diff
@@ -4,50 +4,43 @@
 import android.content.Intent;
 import android.net.Uri;
 import android.os.Bundle;
-import android.support.v4.app.Fragment;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
-import android.util.Log;
-import android.view.View;
-import android.widget.ProgressBar;
-import android.widget.TextView;
 import android.widget.Toast;
 
 import com.tiptopgoodstudio.androidresources.R;
 import com.tiptopgoodstudio.androidresources.Utilities;
-import com.tiptopgoodstudio.androidresources.db.entity.Resource;
 import com.tiptopgoodstudio.androidresources.ui.adapters.ResourceListAdapter;
 import com.tiptopgoodstudio.androidresources.viewmodel.ResourceViewModel;
 
-import java.util.List;
-
 /**
- * Activity displays Android resource list by user-selected topic
+ * Activity displays Android resource list filtered by user-selected topic
  */
 public class ResourceListActivity extends AppCompatActivity
         implements ResourceListAdapter.ResourceClickListener {
 
-    private static final String TAG = ResourceListActivity.class.getSimpleName();
-
     private RecyclerView mRecyclerView;
     private ResourceListAdapter mResourceListAdapter;
-    private ProgressBar mLoadingIndicator;
-    private TextView mErrorMessageDisplay;
     private String mResourceTopic;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.resources_list);
+        setContentView(R.layout.activity_resources_list);
 
-        // progress bar displayed while loading the results
-        mLoadingIndicator = (ProgressBar) findViewById(R.id.pb_loading_indicator);
```

Comment:
What is the reason for removing the progress bar? Will there be a blank screen before the resource list is populated?

Reply:
As long as the data populating the LiveData object is pulling from a local db, the results will be returned so fast that a progress indicator is not needed.  Since the user can't get past the splash screen until there is data in the local db, there is no way this activity could be opened by the user without there being data in the local db.

Reply:
When testing on the emulator, it would take a fraction of a second, with progress bar noticeable during that time. Maybe older phone, or phones with less memory take longer to retrieve data from DB? Is having a progress bar not a recommended practice?

Reply:
The documentation says that you should use one when you do not know how long an operation will take.  If a user has an older phone, everything they do (and all apps which call to a local db) will take a little longer.  My preference for the MVP is to keep things as lean UI-wise as possible and get it out and manually tested.  I feel like we can always add visual extras in later, but I also don't feel strongly about this.

Reply:
I also don't feel strongly about this. Just that most of the exercises in the course involved having a progress bar, so I did not consider it as a visual extra, but almost like a default in every activity. Let's leave this for now, so keep the UI lean.

Reply:
Totally understand (and agree).  I stripped out some of the TextViews which presented error messages simply because, during debugging and trying to get the LiveData to work, it was complete mental overload to have so many view objects which needed to be made visible or invisible under various different conditions.  We can add as much or as little as you want back in.